graylog2-rest-client/src/main/java/org/graylog2/restclient/models/Extractor.java

```diff
@@ -37,6 +38,7 @@
 
     public interface Factory {
         Extractor fromResponse(ExtractorSummaryResponse esr);
+        Extractor fromResponse(ExtractorSummary es);
```

Comment:
Naming the new method `fromExtractorSummary()` would also make it more obvious that `ExtractorSummary` isn't exclusively as a response object.